plugins/homepage/server/__init__.py

```diff
@@ -35,21 +40,45 @@ def __init__(self):
 
     @access.public
     @describeRoute(
-        Description('Public url for getting the homepage markdown.')
+        Description('Public url for getting the homepage properties including markdown content.')
```

Comment:
Since this provides much more than just markdown now, I'd be ok if we renamed this endpoint. It would be an incompatible change, but I'm not sure if this plugin needs to have a stable external API. What do you think @girder/developers ?